backend/app/utils.py

```diff
@@ -410,6 +410,16 @@ def appstream2dict(appstream_url=None) -> dict[str, dict]:
                         and tag.text is not None
                         and tag.text.strip()
                     ]
+
+                    if elem.tag == "keywords" and translated_items:
+                        translated_items = sorted(
```

Comment:
I don't like the idea of sorting, as I would hope, that the most relevant items are first in the keywords. So we would be loosing meaning here. Especially if we at some point decide, that every app only gets a max keyword amount.